app/routes/pages/components/LandingPage.js

```diff
@@ -209,4 +276,33 @@ const committeeEmails = [
   }
 ];
 
+const socialMedia = {
+  facebook: [
+    {
+      link: 'https://www.facebook.com/AbakusNTNU/?ref=bookmarks',
+      textInfo: 'Hovedside for Abakus'
+    },
+    {
+      link:
+        'https://www.facebook.com/Abakus-Bedriftsarrangementer-245005099523282/',
+      textInfo: 'Bedriftsarrangementer'
+    }
+  ],
+  instagram: [
+    {
+      link: 'https://www.instagram.com/abakusntnu/',
+      textInfo: 'Abakus instagram'
+    },
+    {
+      link: 'https://www.instagram.com/abakushs/',
+      textInfo: 'HS instagram'
+    }
+  ],
+  snapchat: [
+    {
+      name: 'abakusntnu'
```

Comment:
Do "@abakusntnu" instead since that's the way I've seen such accounts indicated (and it looks slightly more clear :))